### PR TITLE
Express.js example update

### PR DESCRIPTION
**Pull request by diberry**
* instructions in readme
* /App/data/cache.json - fixes [Error: ENOENT: no such file or directory, open 'C:\repos\code\sdk\microsoft-authentication-library-for-js\samples\msal-node-samples\ExpressTestApp\TestApp\App\data\cache.json']
* Add samples Uris so customer doesn't have to figure out what they are

### CODE REVIEW

**Comment by diberry** [comment]
I would like to make a change to how the TS project and JS project in this sample are related so that the project can easily be deployed to Azure app service. There are a few ways to do that - what do you think?

**Comment by jasonnutter** [comment]
> I would like to make a change to how the TS project and JS project in this sample are related so that the project can easily be deployed to Azure app service. There are a few ways to do that - what do you think?

Definitely open to feedback and ways to make it easier to use!

**Review by derisen — APPROVED** [review]
LGTM!

**Comment by derisen** [comment]
> I would like to make a change to how the TS project and JS project in this sample are related so that the project can easily be deployed to Azure app service. There are a few ways to do that - what do you think?

@diberry sounds good! admittedly neglected the deployment aspect here..

**Comment by diberry** [comment]
@derisen @jasonnutter This is a really awesome sample - I would like to build a tutorial around it to put in the [JS Dev Experience docs](https://docs.microsoft.com/en-us/azure/developer/javascript/). Any issue with that?

**Comment by derisen** [comment]
> @derisen @jasonnutter This is a really awesome sample - I would like to build a tutorial around it to put in the [JS Dev Experience docs](https://docs.microsoft.com/en-us/azure/developer/javascript/). Any issue with that?

@diberry sounds good to me! There are a few improvements I'm planning to work on in the next couple of days e.g. simpler way of using certificates instead of client secrets and using [passport-azure-ad](https://github.com/AzureAD/passport-azure-ad) for token validation etc.

**Comment by diberry** [comment]
#sign-off please merge

File: samples/msal-node-samples/ExpressTestApp/TestApp/appSettings.json [hunk 1]
```diff
@@ -6,8 +6,8 @@
     },
     "settings": {
         "homePageRoute": "/home",
-        "redirectUri": "Enter_the_Redirect_Uri_Here",
-        "postLogoutRedirectUri": "Enter_the_Post_Logout_Redirect_Uri_Here"
+        "redirectUri": "http://localhost:4000/redirect",
```
Comment (sameerag) [inline samples/msal-node-samples/ExpressTestApp/TestApp/appSettings.json:9]: Why are we not leaving this customizable? This mandates the user to have `http://localhost:4000/redirect` as a specific redirectURI to be registered. Is the goal to have a one click working sample?

Comment (sameerag) [reply samples/msal-node-samples/ExpressTestApp/TestApp/appSettings.json:9]: Ah! I see now. We are doing it above but leaving it open here. Never mind.

**Review by sameerag — APPROVED** [review]

File: samples/msal-node-samples/ExpressTestApp/TestApp/appSettings.json (hunk 1 above)
Comment (sameerag) [reply samples/msal-node-samples/ExpressTestApp/TestApp/appSettings.json:9]: Can we also add some comments, which will mention that this is customizable?

Comment (derisen) [reply samples/msal-node-samples/ExpressTestApp/TestApp/appSettings.json:9]: Can't put comments in JSON :( 
README mentions its for example though.

Comment (sameerag) [reply samples/msal-node-samples/ExpressTestApp/TestApp/appSettings.json:9]: Makes sense. Resolving.

**Pull request merged**